app/templates/gentelella/users/mytickets/mytickets_list.html

```diff
@@ -1,11 +1,11 @@
-{% extends 'gentelella/admin/base.html' %}
+{% extends 'gentelella/base.html' %}
 {% block title %}
     {{ page_content.title }}
 {% endblock %}
 {% block head_css %}
     {{ super() }}
-    <link rel="stylesheet" href="{{ url_for('static', filename='css/admin/explore.css') }}">
-    <link href="{{ url_for('static', filename='css/admin/mytickets.css') }}" rel="stylesheet">
+    <link rel="stylesheet" href="{{ url_for('static', filename='css/events/explore.css') }}">
```

Comment:
This file doesn't exist. Maybe this url should be changed to `css/guest/explore/explore.css`